test/danetest.c

```diff
@@ -22,12 +22,16 @@
 #ifndef OPENSSL_NO_ENGINE
 #include <openssl/engine.h>
 #endif
+#include "testutil.h"
+#include "test_main_custom.h"
 
-#include "../e_os.h"
+#include "e_os.h"
```

Comment:
Sadly, I have to change my mind on this.  Not that I have a change of heart, but for technical reasons (sadly, however much I love VMS, there are some quirks in the C compiler that I'm much less happy with...  and I'm saying this in the kindest way possible)